src/main/js/components/Staff.js

```diff
@@ -15,17 +16,33 @@ import "../../resources/static/css/staff.css";
  */
 
 export class Staff extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {pageSize: 30, selectedView: 'Staff'};
+    }
     render() {
         return (
             <div className={"page staff-page"}>
-                <StaffLanding orders={this.props.orders}
```

Comment:
This should not have been deleted. 
Perhaps it was temporary, but staff landing page is the entry point to the staff view, and should remain. 
StaffDiningSessionPage should be redirected to upon click of the `All Orders` button in the StaffLanding page.